wrench/reftests/text/reftest.list

```diff
@@ -34,7 +34,8 @@ fuzzy(1,4) platform(linux) options(disable-subpixel) == colors.yaml colors-alpha
 # Run without dual-source blending path, batches are broken when text colors change.
 fuzzy(1,6) platform(linux) options(disable-dual-source-blending) draw_calls(5) == colors.yaml colors-subpx.png
 # Run with both dual-source blending, ensuring batching is improved.
-fuzzy(1,6) platform(linux) draw_calls(2) == colors.yaml colors-subpx.png
+# TODO: enable when dual source blending is supported
+#fuzzy(1,6) platform(linux) draw_calls(2) == colors.yaml colors-subpx.png
```

Comment:
Note that we expose `DUAL_SRC_BLENDING` flag in `PhysicalDevice::features()` that you can check for

Reply:
Great! Will check it out.